src/VisualStudio/Core/Def/Implementation/PullMemberUp/SymbolDependentsBuilder.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Operations;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.PullMemberUp
+{
+    internal class SymbolDependentsBuilder : OperationWalker
```

Comment:
this type is super weird.  you've made the walker public.  But you expose statics from it that have nothing to do ith it.  The statics instantiate the walker and then use it, but then throw it away and return the results.

Reply:
this comment still applies.